travis.cfg

```diff
@@ -32,6 +32,7 @@ eggs +=
     castle.cms[test]
     celery[sqlalchemy]
     collective.celery
+    collective.documentviewer
```

Comment:
we don't want to commit this change, as documentviewer will still generally be installed as it is currently.

Reply:
Just removed it.